Content.Client/Content.Client.csproj

```diff
@@ -33,6 +33,13 @@
       <DependentUpon>OptionsMenu.cs</DependentUpon>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <Page Include="UserInterface\AdminMenu\CustomControls\ButtonGrid.xaml" />
+    <Page Include="UserInterface\AdminMenu\Tabs\AdminbusTab.xaml" />
+    <Page Include="UserInterface\AdminMenu\Tabs\AdminTab.xaml" />
+    <Page Include="UserInterface\AdminMenu\Tabs\AtmosTab.xaml" />
+    <Page Include="UserInterface\AdminMenu\Tabs\RoundTab.xaml" />
```

Comment:
This doesn't seem intentional?

Reply:
Yea, it was a problem with how I first set up the creation of xaml template.